2019/readme.md

```diff
@@ -0,0 +1,128 @@
+# 2019 File Generation Features
+
+## The FileGenerator Class 
```

Comment:
Please make the same changes for 2019 that are requested in the comments for 2018.

Reply:
This comment might have been made on an outdated version. The version is up to date.